index.bs

```diff
@@ -55,13 +55,46 @@ because, instead of waiting for a new kind of device to be popular enough for
 browsers to provide a specific API, new and innovative hardware can be built
 for the web from day one.
 
+# Motivating Applications # {#motivating-applications}
+
+<em>This section is non-normative</em>.
+
+## Educational Devices ## {#app-edu-devices}
+
+The software delivery model of the web is a key enabler for educational
+applications because they can be quickly loaded on any computer without
+questions of platform compatibility or administrative credentials. Science
+classes are incorporating computerized measurement and data logging into their
+lessons. These tools require bundled software that may be difficult to install
+on managed computers as every new native application adds overhead to an already
+stressed IT department. Web-based hardware APIs will allow support for these
```

Comment:
Maybe switch from future tense to present tense when talking about what WebUSB can do. After all, you're planning to ship in Chrome 60.